consensus/consensus-types/src/vote_proposal.rs

```diff
@@ -51,6 +57,31 @@ impl VoteProposal {
     pub fn next_epoch_state(&self) -> Option<&EpochState> {
         self.next_epoch_state.as_ref()
     }
+
+    pub fn vote_data(&self) -> VoteData {
```

Comment:
better rename to vote_data_ordering_only

Reply:
resolved